app/src/main/res/layout/media_list_item.xml

```diff
@@ -1,96 +1,97 @@
-            app:layout_constraintTop_toTopOf="parent"
-            tools:text="An interesting title" />
+        android:id="@+id/media_title"
+        style="@style/ListItemTitle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:padding="8dp"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="An interesting title" />
 
     <TextView
-            android:id="@+id/date_textview"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:padding="8dp"
-            app:layout_constraintLeft_toRightOf="@id/image_cover"
-            app:layout_constraintTop_toBottomOf="@id/media_title"
-            tools:text="04/25/1989" />
+        android:id="@+id/date_textview"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:padding="8dp"
+        app:layout_constraintLeft_toRightOf="@id/image_cover"
+        app:layout_constraintTop_toBottomOf="@id/media_title"
+        tools:text="04/25/1989" />
 
     <TextView
-            android:id="@+id/media_type"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_margin="8dp"
-            android:text="@string/media_type"
-            app:layout_constraintLeft_toRightOf="@id/image_cover"
-            app:layout_constraintTop_toBottomOf="@id/date_textview" />
+        android:id="@+id/media_type"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_margin="8dp"
+        android:text="@string/media_type"
+        app:layout_constraintLeft_toRightOf="@id/image_cover"
+        app:layout_constraintTop_toBottomOf="@id/date_textview" />
 
     <TextView
-            android:id="@+id/series"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            app:layout_constraintLeft_toLeftOf="@id/date_textview"
-            app:layout_constraintTop_toBottomOf="@id/date_textview" />
+        android:id="@+id/series"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintLeft_toLeftOf="@id/date_textview"
+        app:layout_constraintTop_toBottomOf="@id/date_textview" />
 
     <LinearLayout
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:gravity="end"
-            android:orientation="vertical"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintRight_toRightOf="parent"
-            app:layout_constraintTop_toBottomOf="@id/media_title">
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="16dp"
```

Comment:
This margin is the change for this PR. The rest of the changes are just file reformatting (ctrl+alt+L in Android Studio).